doc/assets/xml_spec.tex

```diff
@@ -164,7 +166,7 @@ \subsection{Trace Steps in XML}
   <location .. />
   <type>signed int</type>
   <full_lhs>b</full_lhs>
-  <full_lhs_value>-1879048192</full_lhs_value>
+  <full_lhs_value binary="10010000000000000000000000000000">-1879048192</full_lhs_value>
```

Comment:
This seems like a bit of a weird way of giving this.  As there are multiple possible representations of the value, wouldn't it be better as `<full_lhs_value_binary></>` `<full_lhs_value_decimal></>` etc.?  Or possibly with `full_lhs_value` be the default human-readable.  Then again, I am not an XML user so maybe this is the appropriate way of doing it.

Reply:
This is consistent with the way that the binary value is already given for floating point values. My intention is to add this feature in a way which will require fewer code changes for anyone who is processing the xml traces. If I was writing this representation from scratch, I would have probably chosen to given equal prominence to the binary and the decimal. This could looks something like -
```
<full_lhs_value><binary>10010000000000000000000000000000</binary><decimal>-1879048192</decimal></full_lhs_value>
```